app/components/add-chat-account-xmpp/component.js

```diff
@@ -0,0 +1,83 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import Space from 'hyperchannel/models/space';
+
+export default class AddChatAccountXmppComponent extends Component {
+
+  @service router;
+  @service coms;
+  @service('sockethub-xmpp') xmpp;
+  @service('remotestorage') storage;
+
+  @tracked username;
+  @tracked host = 'kosmos.org';
+  @tracked password;
+  @tracked connectError = null;
+
+  get userAddress () {
+    return `${this.username}@${this.host}`
+  }
+
+  handleConnectStatus (eventName, message) {
+    console.debug('[add-chat-account-xmpp] handleConnectStatus called') // TODO remove
+
+    if (message.context !== 'xmpp' ||
+        !['message', 'completed'].includes(eventName)) { return; }
+
+    if (message['@type'] === 'error' &&
+        message.object.condition === 'not-authorized'
+        /* && TODO message.actor['@id'] === actor */) {
```

Comment:
It's commented, because of https://github.com/sockethub/sockethub/issues/492,

You are exactly right. It will catch other error messages as well right now. So we need to fix that before using the component after the onboarding later.